### PR TITLE
Added a voting pin for civic-duty minded players.

### PR DESCRIPTION
**Pull request by ShiftyRail**
What it says on the tin. Will need a secret repo update too.

:cl:
- rscadd: Voting on server polls now gives you a neat little pin for the next round you join.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
And by the way, you only get the pin for the very next round you join.

**Comment by toomykins** [comment]
feel like it'd interfere with IC IAA democracy gimmicks

**Comment by ShiftyRail** [comment]
Server polls do not happen nearly that often. And it actually gives the admin something to bus the IAA with.

File: code/modules/client/client defines.dm [hunk 1]
```diff
@@ -94,5 +94,8 @@
 	var/list/seen_messages = list()
 	var/toggle_runechat_outlines = TRUE
 
+	// Voting & civic duty
+	var/ivoted = FALSE
```
Comment (boy2mantwicethefam) [inline code/modules/client/client defines.dm:98]: Technically if you're ready to grapple with how the information is stored you can have it preserve the "I voted" status

Comment (ShiftyRail) [reply code/modules/client/client defines.dm:98]: No. This badge is meant to last for one round only.

**Comment by ShiftyRail** [comment]
Secret repo MR done.

**Pull request merged**